pkgs/top-level/python-packages.nix

```diff
@@ -5330,6 +5330,8 @@ in {
 
   rubymarshal = callPackage ../development/python-modules/rubymarshal { };
 
+  ruffus = callPackage ../development/python-modules/ruffus { };
```

Comment:
Probably didn't mean for that to happen

Reply:
That's what I just realized on irc... :(

Reply:
> > In the course of adding a new package
> 
> Just wanted to mention that if said package requires other deps that you need to package, you can do all of that in one pr.

Thanks, it does; and I wasn't sure how to do that and that makes it a lot easier to do than what I was going to... :)